controls/5_2_ssh_server_configuration.rb

```diff
@@ -186,9 +186,22 @@
     its(:MACs) { should_not be_nil }
   end
 
+  ALLOWED_MACS = [
+    'hmac-sha2-512-etm@openssh.com',
+    'hmac-sha2-256-etm@openssh.com',
+    'umac-128-etm@openssh.com',
+    'hmac-sha2-512',
+    'hmac-sha2-256',
+    'umac-128@openssh.com',
+    'curve25519-sha256@libssh.org',
+    'diffie-hellman-group-exchange-sha256'
+  ].freeze
+
   if sshd_config.MACs
-    describe sshd_config.MACs.split(',').each do
-      it { should match(/^((hmac-sha2-512-etm@openssh\.com|hmac-sha2-256-etm@openssh\.com|umac-128-etm@openssh\.com|hmac-sha2-512|hmac-sha2-256|umac-128@openssh\.com|curve25519-sha256@libssh\.org|diffie-hellman-group-exchange-sha256),)*(hmac-sha2-512-etm@openssh\.com|hmac-sha2-256-etm@openssh\.com|umac-128-etm@openssh\.com|hmac-sha2-512|hmac-sha2-256|umac-128@openssh\.com)$/) }
+    sshd_config.MACs.split(',').each do |m|
```

Comment:
It feels like we re-invent the wheel. We should consider depending on https://github.com/dev-sec/ssh-baseline and the defined resources to define the best macs and ciphers for the operating system.

Reply:
That makes sense, sorry I don't know your overall goals or layout for all of dev-sec just wanting to use the common benchmark for my purposes.

Reply:
@millerthomasj Oh, that was not a complaint :-) I just wanted to mention it, since we should consider harmonizing this in the future

Reply:
I'll merge this for now, and I agree that we should start reusing code in the other profiles. Will have a look at the best way to approach that.